domains/jhedmendoza.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "jhedmendoza",
+        "email": "jhed.mendoza@hotmail.com"
+    },
+    "record": {
+        "CNAME": "jhedmendoza.is-a.dev"
```

Comment:
You cannot put the same domain as `CNAME`

Reply:
I already updated it.

Reply:
> You cannot put the same domain as `CNAME`

Hi, following this up. Thanks.